docs/resources/policy_project.md

```diff
@@ -40,9 +40,9 @@ The following arguments are supported:
 * `default_security_profile`- (Optional) Default security profile properties for project.
     * `north_south_firewall` - (Required) North South firewall configuration.
           * `enabled` - (Required) This flag indicates whether north-south firewall (Gateway Firewall) is enabled. If set to false, then gateway firewall policies will not be enforced on the VPCs associated with this configuration.
-* `vc_folder` - (Optional) Flag to specify whether the DVPGs created for project segments are grouped under a folder on the VC. Defaults to `true`.
-* `quotas` - (Optional) List of policy paths for quota resources that are applicable to this project.
-* `default_span_path` - (Optional) Policy path of the Cluster based default Span object of type NetworkSpan.
+* `vc_folder` - (Optional) Flag to specify whether the DVPGs created for project segments are grouped under a folder on the VC. Defaults to `true`.Available since NSX 9.0.0.
+* `quotas` - (Optional) List of policy paths for quota resources that are applicable to this project.Available since NSX 9.0.0.
+* `default_span_path` - (Optional) Policy path of the Cluster based default Span object of type NetworkSpan.Available since NSX 9.1.0.
```

Comment:
```suggestion
* `default_span_path` - (Optional) Policy path of the Cluster based default Span object of type NetworkSpan. Available since NSX 9.1.0.
```